README-testing.md

```diff
@@ -63,19 +63,93 @@ Where `args.json` looks like this:
 	            ]
 	        }
         ],
-        "selinux": [
-            {"name": "avc_cache_threshold", "value": 512}
-        ],
         "purge": false
     }
 }
 ```
 
-### Other tests
+### tox
+
+Run `tox` by itself with no arguments to run all of the default tests.  Use
+`tox -a` to list all of the test environments.  Use `tox -e name` to run the
+tests in that particular environment.
+
+## Local Virtual Machine based tests
+
+### Setup and running
+
+You must first install the commands
+`/usr/share/ansible/inventory/standard-inventory-qcow2` and `/usr/bin/ansible-playbook`.  On Fedora, this is
+provided by the `standard-test-roles-inventory-qemu` and `ansible` packages -
+`sudo dnf -y install standard-test-roles-inventory-qemu ansible`.
+The test playbooks are the files
+that match the pattern `tests/tests_*.yml`.  You must also have one or more
```

Comment:
it is `tests/tests*.yml` : https://docs.fedoraproject.org/en-US/ci/tests/#_enabling